### PR TITLE
Add postgres user and password properties to config

### PR DESCRIPTION
**Pull request by DanVanAtta**

### CODE REVIEW

File: src/test/java/games/strategy/engine/config/LobbyPropertyReaderTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  private LobbyPropertyReader testObj;
+
+  /**
+   * Sets up an example lobby property file with some fake values, then sets up a property reader test object
+   * pointed at this file.
+   */
+  @Before
+  public void setup() throws IOException {
+    final File testFile = File.createTempFile("testing", "tmp");
+    testFile.deleteOnExit();
```
Comment (ssoloff) [inline src/test/java/games/strategy/engine/config/LobbyPropertyReaderTest.java]: For future reference, the JUnit `TemporaryFolder` rule is a useful alternative to this construct.

File: src/test/java/games/strategy/engine/config/LobbyPropertyReaderTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    try (FileWriter writer = new FileWriter(testFile)) {
+      writer.write(keyValuePair(LobbyPropertyReader.PropertyKeys.port, String.valueOf(TestData.fakePort)));
+      writer.write(keyValuePair(LobbyPropertyReader.PropertyKeys.postgresUser, TestData.fakeUser));
+      writer.write(keyValuePair(LobbyPropertyReader.PropertyKeys.postgresPassword, TestData.fakePassword));
+      writer.close();
```
Comment (ssoloff) [inline src/test/java/games/strategy/engine/config/LobbyPropertyReaderTest.java]: Unnecessary call to `close()` due to try-with-resource.

Comment (DanVanAtta) [inline src/test/java/games/strategy/engine/config/LobbyPropertyReaderTest.java]: Thanks, was left in, now removed.
#codingTooLateAtNight

File: src/test/java/games/strategy/engine/config/LobbyPropertyReaderTest.java (hunk 1 above)
Comment (DanVanAtta) [inline src/test/java/games/strategy/engine/config/LobbyPropertyReaderTest.java]: I looped it in. Looking at the before and after, not terribly convinced in this case, seems marginal. If creating multiple temp folders/files though, then I see it becoming more useful. Thanks for pointing the rule out, I appreciate the reference.

**Pull request merged**